src/utils/constructors/gmx.ts

```diff
@@ -118,3 +127,108 @@ export async function getGmxRegistryConstructorParams(
     (await implementation.populateTransaction.initialize(initializer, core.dolomiteRegistry.address)).data!,
   ];
 }
+
+export async function getGmxRegistryV2ConstructorParams(
+  core: CoreProtocol,
+  implementation: GmxRegistryV2,
+): Promise<any[]> {
+  if (!core.gmxEcosystem) {
+    throw new Error('GMX ecosystem not initialized');
+  }
+
+  const calldata = await implementation.populateTransaction.initialize(
+    core.gmxEcosystem!.gmxExchangeRouter.address,
+    core.gmxEcosystem!.gmxReader.address,
+    core.gmxEcosystem!.gmxRouter.address,
+    core.gmxEcosystem!.gmxDepositHandler.address,
+    core.gmxEcosystem!.gmxDepositVault.address,
+    core.gmxEcosystem!.gmxWithdrawalHandler.address,
+    core.gmxEcosystem!.gmxEthUsdMarketToken.address,
+    core.dolomiteRegistry.address,
+  );
```

Comment:
Make sure this is updated to reflect the ordering after you reorder the contract